AppWidget/glance-widget/build.gradle

```diff
@@ -30,6 +30,9 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
```

Comment:
Isn't this unnecessary if you are replacing everything with Compose?

Reply:
Done.